pyccel/ast/numpyext.py

```diff
@@ -1160,8 +1160,10 @@ def is_elemental(self):
     'complex64' : NumpyComplex64,
     'matmul'    : NumpyMatmul,
     'sum'       : NumpySum,
-    'max'       : NumpyMax,
+#    'max'       : NumpyAmax,
+    'amax'      : NumpyAmax,
     'min'       : NumpyMin,
+    'amin'      : NumpyMin,
```

Comment:
We should use `NumpyAmax`/`NumpyAmin` or `NumpyMax`/`NumpyMin` but not a combination of the 2. Personally I prefer `amax` but it is up to you

Reply:
yes, amax makes more sense for us.